lib/main/atom/onSaveHandler.ts

```diff
@@ -6,7 +6,8 @@ import atomConfig = require('./atomConfig'); ///ts:import:generated
 ///ts:import=parent
 import parent = require('../../worker/parent'); ///ts:import:generated
 
-import {errorView, show} from "./views/mainPanelView";
+import {getFileStatus} from "../atomts";
```

Comment:
It might be a silly question, but why didn't this cyclic dependency cause any problems while I was working on this piece of code? Shouldn't it have crashed Atom or something?